ruby/README.md

```diff
@@ -16,8 +16,117 @@ Alternately, include "rcloadenv" in your bundle.
+so you can the `GOOGLE_APPLICATION_CREDENTIALS` environment variable or
+configure `gcloud auth`.
+
+## Example: Loading the Rails SECRET_KEY_BASE in Google App Engine
+
+This gem is commonly used to provide sensitive information such as API keys or
+database passwords to a Ruby application deployed to Google Cloud Platform,
+without exposing them to configuration files checked into source control.
+
+For example, Rails applications require the environment variable
+`SECRET_KEY_BASE` to be set in the production environment. When deploying a
+Rails application to Google App Engine, you could set this environment
+variable in the "app.yaml" configuration file, but that is not recommended
+because the "app.yaml" file is commonly checked into source control. Instead,
+you can set the `SECRET_KEY_BASE` value securely in the Runtime Config
+service, and use `rcloadenv` to load it into the Rails app. Here's how.
+
+1.  We will assume that you have a [Ruby on Rails](http://rubyonrails.org/)
+    application, you have set up a
+    [Google App Engine](https://cloud.google.com/appengine/) project to deploy
+    it to, you have the [Google Cloud SDK](https://cloud.google.com/sdk/)
+    installed, and you have logged in with gcloud and set your project name in
+    the gcloud configuration.
+
+    See https://cloud.google.com/ruby for more information on deploying a
+    Ruby application to Google App Engine.
+
+2.  Enable the Runtime Config API for your project using the Google Cloud
+    Console (https://console.cloud.google.com/). To do so, navigate to
+    https://console.cloud.google.com/apis/api/runtimeconfig.googleapis.com/overview
+    choose the correct project and click "Enable".
+
+3.  Use the gcloud command line to create a Runtime Configuration:
+
+        gcloud beta runtime-config configs create my-config
+
+    Choose a name for your configuration and replace `my-config` with that
+    name. Any keys you set in this configuration will be loaded as environment
+    variables in your application.
+
+    Because you will be storing sensitive information in this configuration
+    resource, you may consider restricting access to it. See
+    https://cloud.google.com/deployment-manager/runtime-configurator/access-control
+    for more information. If you do so, make sure any service accounts that
+    run `rcloadenv` retain access to the resource. That includes the App Engine
+    service account (which runs your application in App Engine) and the
+    Cloud Build service account (which performs build steps such as asset
+    precompilation for your application).
+
+4.  Create a secret key
+
+        bundle exec rake secret
+
+    This will generate a random key and print it to the console.
+
+5.  Use the gcloud command line to set `SECRET_KEY_BASE` in your configuration.
+
+        gcloud beta runtime-config configs variables set \
+          SECRET_KEY_BASE 12345678 --config-name=my-config
```

Comment:
You're right that we should show `--is-text`. It turns out, however, that the error you're seeing is actually due to YAML parsing on the Rails side: if the value is all digits, it gets written as such in secrets.yml, and YAML parses it as an integer. This "never" happens in practice because the chance of `rake secret` generating such a value is infinitesimal. But to avoid confusion, I changed the example value.